scripts/exportexcel.py

```
@@ -0,0 +1,97 @@
+    sys.path.append(current_dir)
+
+from src.jsonsync import JsonItem
+
+
+def prettify_json(to_prettify):
+    return ', '.join([f"{key}={val}" for key, val in json.loads(to_prettify).items()])
+
+current_dir = Path(os.getcwd())
+
+if os.path.basename(current_dir.resolve()) != "clp-bench":
+    raise Exception("Must be run in clp-bench directory, if it was renamed, comment this line out")
```

Ruff (0.12.2)

Create your own exception

(TRY002)